service/posts/pagination.py

```diff
@@ -10,6 +11,15 @@ class PostsPagination(pagination.PageNumberPagination):
     page_size_query_param = "size"
 
     def get_paginated_response(self, data):
+        for post in data:
+            # Some groups end source in "/" some do not, we handle both cases.
+            if post["source"][-1] == "/":
+                post["next"] = post["source"] + "comments/"
+            else:
+                post["next"] = post["source"] + "/comments/"
+            post["count"] = len(Comment.objects.filter(post_id=post["id"]))
```

Comment:
Using [`count()`](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#count) instead of `len()` will be more efficient.